recipes/nkdfu/LICENSE

```diff
@@ -0,0 +1,339 @@
+		    GNU GENERAL PUBLIC LICENSE
```

Comment:
This file is in the tarball. You do no need to add it here too.

Reply:
Our text "check the license" is not clear. We should improve that. The issue here is, we need a license if that is missing, but if it is present, adding one may keep it out of sync with the one in the tarball as the package evolves. When present,, the one in the tarball should always be used.